Tests/test_trie.py

```diff
@@ -129,6 +129,36 @@ def test_with_prefix(self):
         self.assertEqual(set(['ANA', 'ANANA']),
                          set(trieobj.with_prefix("AN")))
 
+    def test_large_save_load(self):
+        """
```

Comment:
See PEP257 https://www.python.org/dev/peps/pep-0257/

The docstring should be something like this with a title line then a blank line:

``` python
    def test_large_save_load(self):
        """Generate random key/val pairs in three length categories,

        Uses 100 in each category. Insert them into a trie and into a reference dict.
        Write the trie to a temp file and read it back, verify that trie entries match
        the reference dict.
        """
```

The unittest module will also pull out the title line and print it at the command line when running ``test_trie.py`` directly (at least on recent versions of Python).